src/test/java/com/google/javascript/clutz/MultiFileTest.java

```diff
@@ -83,6 +83,15 @@ public void skipEmitWithTypedefs() throws Exception {
         .generatesDeclarations(expected);
   }
 
+  @Test
+  public void multifilePartial() throws Exception {
+    String expected = DeclarationGeneratorTests.getTestFileText(input("singleton.d.ts"));
```

Comment:
nitpicing on the name: 'singleton' in a java code base might bring unwanted associations. How about 'total', that's my new word for old clutz mode.

Reply:
Huh - what does singleton mean in java?

And changed.